include/okapi/api/chassis/model/chassisModel.hpp

```diff
@@ -126,83 +126,31 @@ class ChassisModel : public ReadOnlyChassisModel {
   virtual void setGearing(AbstractMotor::gearset gearset) = 0;
 
   /**
-   * Sets new PID constants.
+   * Sets a new maximum velocity in RPM [0-600].
```

Comment:
How do we know the velocity range is [0-600]? Does okapi normalize it? Are we just taking the value of the largest possible gearset? If so, it might be better to mention that the RPM range depends on the gearset.

Reply:
It's RPM. The maximum does vary based on the gearset, yes, so if you set `600` max but actually have `200` max and tell the chassis to go full velocity, it will try to go `600`. That could be mentioned in the docs.

Reply:
I think it may be confusing to people, and they will assume that regardless of their actual gearset the range will be `[0, 600]`. It's not informative anyways, because it assumes people will already know what their gearset is, disregarding what the comment says.
I think that should be taken out entirely, and replaced with either of the following:
```cpp
/**
 * Sets a new maximum velocity in RPM, according to the gearset.
 * Sets a new maximum velocity in RPM.
```